central/complianceoperator/v2/compliancemanager/manager.go

```diff
@@ -1,14 +1,22 @@
 package compliancemanager
 
-import "context"
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/storage"
+)
 
 // Manager provides functionality to manage compliance requests.
 // Use Manager to process compliance requests and forward them to Sensor.
+//
+//go:generate mockgen-wrapper
 type Manager interface {
 	// Sync reconclies the compliance scan configurations stored in Central with all Sensors.
 	// Use it to sync scan configurations upon Central start, Sensor start, and ad-hoc sync requests.
 	Sync(ctx context.Context)
 
+	ProcessComplianceOperatorInfo(ctx context.Context, complianceIntegration *storage.ComplianceIntegration) error
```

Comment:
Could you add a comment? I was a bit confused about whether `ProcessComplianceOperatorInfo` is processing the message read from the database or coming from the sensor.